contractcourt/channel_arbitrator.go

```diff
@@ -325,6 +322,11 @@ type ChannelArbitrator struct {
 	// to do its duty.
 	cfg ChannelArbitratorConfig
 
+	// blocks is a channel that the arbitrator will receive new blocks on.
+	// This channel should be buffered by so that it does not block the
+	// sender.
+	blocks chan int32
```

Comment:
Nice, this'll also make it _slightly_ easier to write unit tests in this area as well1